src/scripts/setup_external_storage/setup_external_storage.py

```diff
@@ -0,0 +1,61 @@
+"""
+Create an STS-enabled folder on Synapse over an S3 location.
+"""
+
+import os
+import json
+import argparse
+
+import synapseclient
+
+
+def read_args():
+    parser = argparse.ArgumentParser(
+        description="Create an STS-enabled folder on Synapse over an S3 location."
+    )
+    parser.add_argument(
+        "--synapse-parent", help="Synapse ID of the parent folder/project"
+    )
+    parser.add_argument(
+        "--synapse-folder-name",
+        help=(
+            "Name of the Synapse folder to use with the external " "storage location."
+        ),
+    )
+    parser.add_argument("--s3-bucket", help="S3 bucket name")
+    parser.add_argument(
+        "--s3-key",
+        default=None,
+        help="S3 key to serve as the root. `/` (root) by default.",
```

Comment:
We know where the production data will be in S3, so let's set a more useful default than the bucket root.